Created new controller for deposits in LimitedSupplyToken

valeriagrazzini opened the pull request:
- Created controller
- Created new method in the DepositService
- Created new routes

peterpolman commented on src/services/AssetPoolService.ts:
```diff
@@ -68,22 +68,23 @@ export default class AssetPoolService {
-                assetPool.contract.methods.deposit(adminBalance),
-                assetPool.network,
-            );
+            const adminBalance = await contract.methods.balanceOf(admin.address).call();
```
@valeriagrazzini I guess this could be removed now right? Since the user should be able to deposit an x amount manually. 

Removing it will speed up pool creation quite a bit:)

peterpolman commented on src/services/DepositService.ts:
```diff
@@ -51,9 +63,49 @@ async function create(assetPool: TAssetPool, deposit: DepositDocument, call: str
 
             return await deposit.save();
         } catch (error) {
+            console.log('ERROR ON CREATE DEPOSIT', error.message)
```
@valeriagrazzini Could be removed or should use the default logger instance.

peterpolman commented:
@valeriagrazzini Made some changes before merging, please have a look at the last 2 commits and ping me if changes are not clear or if you don't agree!

valeriagrazzini commented:
I Peter, i saw some changes yesterday and fixed the gap with the dashboard.
Now i am implementing the list in vue and fixed some errors as well

Il ven 20 mag 2022, 19:26 Peter Polman ***@***.***> ha
scritto:

> @valeriagrazzini <https://github.com/valeriagrazzini> Made some changes
> before merging, please have a look at the last 2 commits and ping me if
> changes are not clear or if you don't agree!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/thxprotocol/api/pull/291#issuecomment-1133147315>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AILICDFTW7FJNU3IOA2ZKILVK7DMFANCNFSM5WGATBZQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
